src/components/views/rooms/ReadReceiptGroup.tsx

```diff
@@ -284,8 +284,7 @@ interface ISectionHeaderProps {
 }
 
 function SectionHeader({ className, children }: PropsWithChildren<ISectionHeaderProps>): JSX.Element {
-    const ref = useRef<HTMLHeadingElement>(null);
-    const [onFocus] = useRovingTabIndex(ref);
+    const [onFocus, , ref] = useRovingTabIndex<HTMLHeadingElement>();
```

Comment:
I don't really understand why this is now required?

Reply:
Strictly speaking it isn't, it is the only call site which didn't conform to using the `ref` returned by `useRovingTabIndex` though